### PR TITLE
Dont optimize without indices

### LINKED ISSUE #154: gltfPrimitiveToCesiumGeometry assumes that indices is defined
Running a gltf model without indices through the pipeline crashes currently at the `cacheOptimization` stage.

### PR DESCRIPTION
**Pull request by lasalvavida**
Fixes #154, @lilleyse, could you look at this when you get a chance?

### CODE REVIEW

File: specs/lib/cacheOptimizationSpec.js [hunk 1]
```diff
@@ -59,4 +59,24 @@ describe('cacheOptimization', function() {
...
+                                NORMAL : 'notImportantEither'
+                            }
+                        }
+                    ]
+                }
+            }
+        };
+        expect(function() {
+            cacheOptimization(gltf);
+        }).not.toThrowRuntimeError();
```
Comment (lasalvavida) [inline specs/lib/cacheOptimizationSpec.js]: I just checked and this doesn't fail without the changes, so you're right, we need a better test here.

**Comment by lasalvavida** [comment]
Updated

File: specs/lib/cacheOptimizationSpec.js [hunk 2]
```diff
@@ -59,4 +59,22 @@ describe('cacheOptimization', function() {
...
+                            attributes : {
+                                POSITION : 'notImportant',
+                                NORMAL : 'notImportantEither'
+                            }
+                        }
+                    ]
+                }
+            }
+        };
+        cacheOptimization(gltf);
```
Comment (lilleyse) [inline specs/lib/cacheOptimizationSpec.js]: You should have at least on expect in here. Maybe just create a spy for `GeometryPipeline.reorderForPostVertexCache` and check that it's not called.

Comment (lilleyse) [inline specs/lib/cacheOptimizationSpec.js]: Oh well I guess that would pass for both...

Comment (lasalvavida) [inline specs/lib/cacheOptimizationSpec.js]: No, that should work. gltfPrimitiveToCesiumGeometry would work too.

**Comment by lasalvavida** [comment]
Updated

**Comment by lilleyse** [comment]
Sorry just had a last minute thought, can you update CHANGES.md with the change? We should start getting in the habit of this now.

**Comment by lasalvavida** [comment]
Yup

**Pull request merged**